ha/util/health_generator/mock_health_event_publisher.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+"""Module helps in generating the mock health event and publishing it to the
+message bus.
+"""
+
+import argparse
+import sys
+import pathlib
+import errno
+import json
+import time
+
+from cortx.utils.conf_store import Conf
+from ha.util.conf_store import ConftStoreSearch
+from ha.core.config.config_manager import ConfigManager
+from ha import const
+from ha.util.message_bus import MessageBus
+from ha.fault_tolerance.event import Event
```

Comment:
Still using Event class of HA? This is going to be removed with PR https://github.com/Seagate/cortx-ha/pull/662.
Can we use health event from utils? @indrajitzagade

Reply:
The utils PR has just been merged, let me update this.

Reply:
Done @mariyappanp